docs/releasing-files.md

```diff
@@ -292,6 +308,14 @@ These outputs can be shared with project collaborators and published in line wit
     - transcribing (e.g., to paper or email)
     - using screen sharing software or any recording device/software
 
+### Running further analyses on released outputs
+
+If you have had [intermediate data released](#release-of-intermediate-data) and you wish to run further analyses on them, such as reformatting figures, there are a few things to consider.
+
+1. You should include the code for these steps in your GitHub repo.
+2. You **should not** commit any of the released outputs to your GitHub repo. Make sure to include them in the `.gitignore` file.
```

Comment:
what about the final processed charts/tables created as a result, should they be excluded from the repo as well?

Reply:
I think they have to be, for now, given our approaches to making repos public and the template readme content. Ultimately, we'll need to be able to upload results to the jobs site